tactics/tacticals.ml

```diff
@@ -734,13 +732,11 @@ module New = struct
   let case_nodep_then_using =
     general_elim_then_using gl_make_case_nodep false
 
-  let pf_constr_of_global ref tac =
-    Proofview.Goal.enter { enter = begin fun gl ->
-      let env = Proofview.Goal.env gl in
-      let sigma = Tacmach.New.project gl in
-      let (sigma, c) = Evd.fresh_global env sigma ref in
-      let c = EConstr.of_constr c in
-      Proofview.Unsafe.tclEVARS sigma <*> (tac c)
-    end }
+  let pf_constr_of_global ref =
```

Comment:
`Evd.fresh_global` creates fresh universes to make an instance for the global (if it's a polymorphic constant) and adds them to the evar map to keep track of them.

I think the semantics do change because previously you would get a fresh instance per goal but now every goal shares the same instance.